code/modules/urist/modules/wfc/modules/deepmaint/teleporters.dm

```diff
@@ -0,0 +1,143 @@
+
+/proc/deepmaint_yeet_trg(var/atom/movable/trg, var/xoffset = 1, var/yoffset = 1)
+	if(!istype(trg, /atom/movable))
```

Comment:
Maybe, but I find it much less readable. You need to mentally disambiguate between the is-implicit-type case and the is-SOME-type case. Because we cannot have sane APIs in BYOND.

Reply:
Fair. I think it's personal preference. For _me_ `istype(trg)` tells me I'm checking if what I have is what I expected, whereas `istype(trg, /atom/movable)` is checking if it's a specific child type of what we currently have.

Reply:
_This could also be just replaced with the built-in `ismovable(trg)`_